embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/arch/destination/LogEventData.kt

```diff
@@ -1,12 +1,14 @@
 package io.embrace.android.embracesdk.arch.destination
 
+import io.opentelemetry.api.logs.Severity
+
 /**
  * Represents a Log event that can be added to the current session span.
  */
 internal class LogEventData(
-    val startTimeNanos: Long,
-    val severityNumber: Int,
-    val severityText: String?,
+    val timestampNanos: Long,
+    val severity: Severity,
```

Comment:
What does a null severity mean? Is this something we currently support?

Reply:
`severityText` is nullable because it it an optional alternative text description. If it's null, the log record will use the default name for the severity passed. The severity itself is not nullable.